backend-api/src/functions/asyncIssueBiometricCredential/getBiometricSession/getBiometricSession.ts

```diff
@@ -0,0 +1,155 @@
+import { logger } from "../../common/logging/logger";
+import { LogMessage } from "../../common/logging/LogMessage";
+import {
+  HttpError,
+  ISendHttpRequest,
+  RetryConfig,
+  sendHttpRequest as sendHttpRequestDefault,
+  SuccessfulHttpResponse,
+} from "../../adapters/http/sendHttpRequest";
+import { Result, successResult, errorResult } from "../../utils/result";
+
+export interface BiometricSession {
+  id: string;
+  finish: string;
+}
+
+export interface GetBiometricSessionError {
+  statusCode?: number;
+  message: string;
+  isRetryable: boolean;
+}
+
+export type GetBiometricSession = (
+  baseUrl: string,
+  biometricSessionId: string,
+  viewerKey: string,
+  sendHttpRequest?: ISendHttpRequest,
+) => Promise<Result<BiometricSession, GetBiometricSessionError>>;
+
+export const getBiometricSession: GetBiometricSession = async (
+  baseUrl: string,
+  biometricSessionId: string,
+  viewerKey: string,
+  sendHttpRequest: ISendHttpRequest = sendHttpRequestDefault,
+) => {
+  const httpRequest = {
+    url: `${baseUrl}/odata/v1/ODataServlet/Sessions('${biometricSessionId}')`,
+    method: "GET" as const,
+    headers: {
+      "X-Innovalor-Authorization": viewerKey,
+      Accept: "application/json;odata.metadata=minimal",
+      "Content-Type": "application/json;odata.metadata=minimal",
+      "OData-MaxVersion": "4.0",
+      "OData-Version": "4.0",
+    },
+  };
+
+  const retryConfig: RetryConfig = {
+    retryableStatusCodes: [
+      429, 500, 501, 502, 503, 504, 505, 506, 507, 508, 510, 511,
+    ],
+    delayInMillis: 50,
+  };
+
+  logger.debug(LogMessage.ISSUE_BIOMETRIC_CREDENTIAL_GET_FROM_READID_ATTEMPT, {
+    data: {
+      biometricSessionId,
+    },
+  });
+
+  const getBiometricSessionResult: Result<SuccessfulHttpResponse, HttpError> =
+    await sendHttpRequest(httpRequest, retryConfig);
+
+  if (getBiometricSessionResult.isError) {
+    const error = getBiometricSessionResult.value;
+    const statusCode = error.statusCode ? Number(error.statusCode) : undefined;
+    const retryableStatusCodes = retryConfig.retryableStatusCodes;
+
+    const isRetryable =
+      !statusCode || (retryableStatusCodes?.includes(statusCode) ?? false);
```

Comment:
```suggestion
      !statusCode || (retryableStatusCodes.includes(statusCode));
```

^ (we know that `retryableStatusCodes` is an array, and `includes` will always return a boolean)